meta-iotqa/conf/test/refkit-image-common-extensive.manifest

```diff
@@ -1 +1,3 @@
 # Extensive tests for common profile
+oeqa.runtime.bluetooth.comm_bt_command
```

Comment:
@mythi, would it be ok then to enable bluetoothctl for the generated images in the CI? If that's the case, where is the best location to set the REFKIT_IMAGE_EXTRA_INSTALL variable? Also, considering that these tests take less than 1 minute, but they will start adding up with other tests, you still would like me to move them to regular sanity testing manifests? How much total time for runtime test execution per profile and platform would you consider the ideal at this time?